src/components/Settings/AllSettingsFields.tsx

```diff
@@ -269,6 +270,9 @@ export const AllSettingsFields = forwardRef(
                 Want to experience the latest and (hopefully) greatest from our
                 main development branch?{' '}
                 <a
+                  onClick={openExternalBrowserIfDesktop(
+                    'https://zoo.dev/modeling-app/download/nightly'
+                  )}
```

Comment:
@franknoirot I've seen this done for the bottom right buttons so I'm replicating it here and it seems to work. Should this be done for the other about links above?

Reply:
Yeah I think it should, good catch!

Reply:
Ok will add!

Reply:
Done